docs/guides/misc/client_errors.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: Known Client Errors
+---
+
+Documentation for various known errors in the client.
+
+## store_list
+
+### STORE ERROR: account is too new
+
+New accounts may only make purchases via ((%Cstore_list%)) after 3 days.
+
+### STORE ERROR: unknown
+
+This error is typically indicative of an untrusted Steam account. Unfortunately, the reason for this error is often difficult to dianogse.
+
+- The Steam account usually must exist for at least 30 days
+- The Steam account needs to have purchased at least $5 of things on Steam, from a real credit/debit card
```

Comment:
This doesn't have to be a credit/debit card, using a different payment method seems to work fine as well (in my case PayPal, I _believe_ I was also able to get by using just Gift Cards? But I'm not sure about that part of it.)

Reply:
I'll change to "a valid payment method"